### PR TITLE
update PrepaidAggregator to use Solidity v0.6

### PR DESCRIPTION
**Pull request by se3000**
Moves all related contracts over when possible, and if solidity is used by other v0.5 contracts then I copied it over instead.

### CODE REVIEW

File: tools/docker/docker-compose.paritynet.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ services:
       - devnet
   devnet:
     container_name: parity
-    image: smartcontract/devnet
+    image: smartcontract/devnet:st-peters
```
Comment (se3000) [inline tools/docker/docker-compose.paritynet.yaml]: Temporary change until we merge https://github.com/smartcontractkit/devnet/pull/43 and release a new latest version.

File: evm-contracts/src/v0.6/Median.sol [hunk 2]
```diff
@@ -0,0 +1,108 @@
+pragma solidity ^0.6.2;
```
Comment (thodges-gh) [inline evm-contracts/src/v0.6/Median.sol]:
```suggestion
pragma solidity ^0.6.0;
```

File: evm-contracts/src/v0.6/dev/AggregatorInterface.sol [hunk 3]
```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.6.2;
```
Comment (thodges-gh) [inline evm-contracts/src/v0.6/dev/AggregatorInterface.sol]:
```suggestion
pragma solidity ^0.6.0;
```

File: evm-contracts/src/v0.6/dev/Owned.sol [hunk 4]
```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.5.0;
+pragma solidity 0.6.2;
```
Comment (thodges-gh) [inline evm-contracts/src/v0.6/dev/Owned.sol]:
```suggestion
pragma solidity ^0.6.0;
```

File: evm-contracts/src/v0.6/dev/SafeMath128.sol [hunk 5]
```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.0;
+pragma solidity ^0.6.2;
```
Comment (thodges-gh) [inline evm-contracts/src/v0.6/dev/SafeMath128.sol]:
```suggestion
pragma solidity ^0.6.0;
```

File: evm-contracts/src/v0.6/dev/SafeMath32.sol [hunk 6]
```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.0;
+pragma solidity ^0.6.2;
```
Comment (thodges-gh) [inline evm-contracts/src/v0.6/dev/SafeMath32.sol]:
```suggestion
pragma solidity ^0.6.0;
```

File: evm-contracts/src/v0.6/dev/SafeMath64.sol [hunk 7]
```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.0;
+pragma solidity ^0.6.2;
```
Comment (thodges-gh) [inline evm-contracts/src/v0.6/dev/SafeMath64.sol]:
```suggestion
pragma solidity ^0.6.0;
```

File: evm-contracts/src/v0.6/dev/Whitelisted.sol [hunk 8]
```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.5.0;
+pragma solidity 0.6.2;
```
Comment (thodges-gh) [inline evm-contracts/src/v0.6/dev/Whitelisted.sol]:
```suggestion
pragma solidity ^0.6.0;
```

File: evm-contracts/src/v0.6/tests/MedianTestHelper.sol [hunk 9]
```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.6.2;
```
Comment (thodges-gh) [inline evm-contracts/src/v0.6/tests/MedianTestHelper.sol]:
```suggestion
pragma solidity ^0.6.0;
```

File: evm-contracts/src/v0.6/tests/OwnedTestHelper.sol [hunk 10]
```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.5.0;
+pragma solidity 0.6.2;
```
Comment (thodges-gh) [inline evm-contracts/src/v0.6/tests/OwnedTestHelper.sol]:
```suggestion
pragma solidity ^0.6.0;
```

File: evm-contracts/src/v0.6/vendor/SignedSafeMath.sol [hunk 11]
```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.6.2;
```
Comment (thodges-gh) [inline evm-contracts/src/v0.6/vendor/SignedSafeMath.sol]:
```suggestion
pragma solidity ^0.6.0;
```

File: integration/common [hunk 12]
```diff
@@ -80,6 +80,13 @@ deploy_v05_contracts() {
   export LINK_TOKEN_ADDRESS=`cat log | grep 'Deployed LinkToken at:' | awk '{print$4}'`
   export COORDINATOR_ADDRESS=`cat log | grep 'Deployed Coordinator at:' | awk '{print$4}'`
   export MEAN_AGGREGATOR_ADDRESS=`cat log | grep 'Deployed MeanAggregator at:' | awk '{print$4}'`
+}
+
+deploy_v06_contracts() {
+  title "Migrating v0.6 contracts..."
+  local log=$LOG_PATH/deployv0.5.log
```
Comment (lorenzb) [inline integration/common]: Should that log path be readjusted?

**Pull request merged**